scripts/install-chef-suse.sh

```diff
@@ -130,7 +131,7 @@ pipe_show_and_log () {
         rm -f $t
         dialog --clear >&3
     fi
-    tee -a /dev/fd/3 /dev/fd/4 > /dev/null
+    tee -a /dev/fd/4 >&3
```

Comment:
I don't know who wrote that dark magic, but he should be able to find out.
I cannot explain that, sorry. the idea came by @bmwiedemann